server.go

```diff
@@ -92,10 +92,12 @@ type Server struct {
 	databases map[string]*database // databases by name
 	shards    map[uint64]*Shard    // shards by id
 	users     map[string]*User     // user by name
+
+	version string // current version of the server
 }
 
 // NewServer returns a new instance of Server.
-func NewServer() *Server {
+func NewServer(version string) *Server {
```

Comment:
I'm open to another approach.  I asked in slack but everyone was busy, but i knew you would see it here :-).